docs/DeveloperGuide.md

```diff
@@ -6,8 +6,248 @@
+  provided `Satisfaction` object.
+
+Given below is an example usage scenario and how the mechanism for adding a satisfaction behaves at each step.
+
+Step 1. The user launches the application. In the `Duke` class, an empty instance of the `SatisfactionList` class,
+called `satisfactionList`, is created.
+
+![Step 1 Object Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_Step1ObjectDiagram.png)
+
+
+Step 2. The user types the command `add satisfaction Bob 5`. In the `Duke` class, a `Command` object
+is created by invoking the `CommandParser` class's constructor on the user input. The details of this
+step are further described below.
+
+![Step 1 Object Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_Step2ObjectDiagram.png)
+
+
+Step 3. The `CommandParser` class replaces the `add satisfaction` in the user input with an empty string,
+leaving just `Bob 5`. Then, the `AddSatisfactionCommand` class's constructor is invoked with `Bob 5`.
+
+![Step 1 Object Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_Step3ObjectDiagram.png)
+
+Step 4. The `AddSatisfactionCommand` class's constructor parses `Bob 5`, extracting the customer name `Bob` using the
+`AddSatisfactionCommand#extractCustomerName(String userInput)` method and extracting the customer satisfaction rating
+`5`using the `AddSatisfactionCommand#extractSatisfactionValue(String userInput)` method. The `AddSatisfactionCommand`
+class constructor then invokes the `Satisfaction` class's constructor to create a new `Satisfaction` object, passing in
+customer name `Bob` and satisfaction rating `5`.
+
+![Step 1 Object Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_Step4ObjectDiagram.png)
+
+Step 5. When the `AddSatisfactionCommand` object is executed in the `Duke` class, the `Satisfaction` object
+created in the `AddSatisfactionCommand` class is added to `satisfactionList`.
+
+![Step 1 Object Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_Step5ObjectDiagram.png)
+
+The following sequence diagram shows what would happen if the user typed `add satisfaction Bob 5`.
+
+PLEASE NOTE: Due to the limitation of PlantUML, the lengths of the activation bars may not be correct, and
+certain return values from method calls that could not be placed correctly were omitted. For example, the
+instantiation of objects of type `SatisfactionList`, `CommandParser`, `Command`, `AddSatisfactionCommand`,
+and `Satisfaction` should have return value arrows (representing the object that was instantiated) to the
+entity that created those instances, but those are not indicated in the diagram. Furthermore, for neatness, the
+`User` entity should be the left-most entity, but that could not be formatted properly in the diagram.
+
+![Sequence Diagram](team/alinazheng_addsatisfactioncommand_uml/AlinaZheng_AddSatisfaction_SequenceDiagram.png)
```

Comment:
![image](https://user-images.githubusercontent.com/69459230/160963383-8d9ca853-8012-410d-8928-61e1e9aae017.png)

Consider changing the arrow from `Duke` to `CommandParser`  to be a solid arrow rather than a dotted arrow as you are making a method call rather than returning from a method call.
Additionally, consider adding a `:` to all class/object entities in the sequence diagram, e.g. `:CommandParser` to follow the correct formatting.

Reply:
![image](https://user-images.githubusercontent.com/69459230/160965031-2e1129cf-5c25-4cec-9696-9e0cd4b941ad.png)

Additionally, consider removing the entities at the bottom as well as showing the ends of the activation bars.